tests/testcases/core/libraries/rest_api/controllers/Read_Test.php

```diff
@@ -1072,6 +1072,42 @@ public function testHandleRequestGetAllVenues()
         $this->assertEquals($others_private_venue->ID(), $data[1]['VNU_ID']);
     }
 
+    /**
+     * Reproduces https://github.com/eventespresso/event-espresso-core/issues/845, which has been an often-recurring
+     * regression.
+     * @since $VID:$
+     */
+    public function testHandleRequestGeAllTermRelationships()
+    {
+        $e = $this->new_model_obj_with_dependencies(
+            'Event',
+            array(
+                'status' => EEM_Event::post_status_publish
+            )
```

Comment:
just as an FYI,
because our min php version is now at 5.4 you can use the short array syntax **if you prefer**,

ie:

```php
       $e = $this->new_model_obj_with_dependencies(
            'Event',
            ['status' => EEM_Event::post_status_publish]
        );
```

Reply:
yes, yes I would. Good point thanks.